bindings/python/test/flight/test_maneuver.py

```diff
@@ -13,6 +13,14 @@
 
 from ostk.astrodynamics.flight import Maneuver
 from ostk.astrodynamics.dynamics import Tabulated as TabulatedDynamics
+from ostk.astrodynamics.trajectory.state import CoordinateSubset
+
+
+from ..dynamics.test_tabulated import (
+    instants as tabulated_instants,
+    contribution_profile as tabulated_contribution_profile,
+    coordinate_subsets as tabulated_coordinate_subsets,
+)
```

Comment:
I wanted to not reimport them because they were just used to test a small part of the class (like one method) instead of a significant part. But yes I think we should move using a conftest or a fixtures folder in the future with ostk cause it'll be cleaner

Reply:
@apaletta3 can you make a github issue for this so we don't forget?